charts/fybrik/templates/manager-deployment.yaml

```diff
@@ -42,11 +42,11 @@ spec:
           command:
           - cp 
           - -rf
-          - "/data/tools"
-          - "/tmp/solver/"
+          - {{ include "fybrik.printDataDir" . }}/tools
+          - {{ include "fybrik.printDataSubdir" ( tuple "solver" ) }}/
```

Comment:
so solver dir will be `/data/solver` instead of `/tmp/solver`. is it correct ?

Reply:
yes its correct.